servers/src/test/java/tachyon/master/lineage/LineageMasterTest.java

```diff
@@ -74,11 +85,121 @@ public void listLineagesTest() throws Exception {
   }
 
   @Test
-  public void createLineageWithNonExistingFileTest()  throws Exception {
-    mThrown.expect(InvalidPathException.class);
-    mThrown.expectMessage(ExceptionMessage.LINEAGE_INPUT_FILE_NOT_EXIST.getMessage("/test1"));
+  public void createLineageWithNonExistingFileTest() throws Exception {
+    TachyonURI missingInput = new TachyonURI("/test1");
+    Mockito.when(mFileSystemMaster.getFileId(missingInput)).thenReturn(-1L);
+    // try catch block used because ExpectedExceptionRule conflicts with Powermock
```

Comment:
I think many people `ExpectedException` because it's more readable. Ideally a style check rule can check `PowerMockRunner` rule does not coexist with `PowerMockRunner`. Tracked at TACHYON-1253